### DISCUSSION TITLE
Zoom In with "Ctrl +" not working as expected on Windows 10

### DISCUSSION THREAD

**Discussion by ManCaveMedia** [f1multiviewer/issue-tracker#157] [Feature requests]
On Windows 10 it's possible to Zoom Out with "Ctrl -", but to Zoom In, you need to press "Ctrl Shift +".
That's probably because the "+" character is actually above the "=" character.

Edit: To add: Ctrl and "+" or "-" on the num pad don't work at all.

**Comment by jeppeolesen** [comment]
Hey there, if this is still relevant, please submit a bug report over here: https://github.com/f1multiviewer/issue-tracker/issues
